src/shared/varlink-systemd.c

```diff
@@ -0,0 +1,21 @@
+/* SPDX-License-Identifier: LGPL-2.1-or-later */
+
+#include "alloc-util.h"
+#include "string-util.h"
+#include "varlink-systemd.h"
+#include "version.h"
+
+int varlink_set_info_systemd(sd_varlink_server *server) {
```

Comment:
Why not place this in `varlink-util`? `-util.[ch]` is where helpers for internal use usually lies? E.g. `bus-util`, `device-util`, `json-util`, etc.